docs/app/css/style.css

```diff
@@ -185,10 +185,6 @@ pre {
 pre > code.highlight {
   padding: 16px;
   font-weight: 400;
-  -webkit-user-select: initial;
-  -moz-user-select: initial;
-  -ms-user-select: initial;
```

Comment:
`-webkit` and `-moz` prefixes can stay since IE ignores them.

This can be changed to `-ms-user-select: text` without needing to remove anything else.

Unless there's a reason to removing the entire code block?